components/script/dom/bluetooth.rs

```diff
@@ -44,13 +47,19 @@ const OPTIONS_ERROR: &'static str = "Fields of 'options' conflict with each othe
 pub struct Bluetooth {
     reflector_: Reflector,
     device_instance_map: DOMRefCell<HashMap<String, MutHeap<JS<BluetoothDevice>>>>,
```

Comment:
this patch shoul be squashed with [this](https://github.com/szeged/servo/pull/74/commits/a71ae7bc55ad2d5f43996ab5beaaf17a4e66a028)